[2020/07/28 19:34:24.901923]========== Start ==========
[2020/07/28 19:34:24.905941][I][main] Python 3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 23:09:28) [MSC v.1916 64 bit (AMD64)]
[2020/07/28 19:34:24.908941][I][main] GFI20LastScript.py 3.0.0.6
[2020/07/28 19:34:24.911951][I][readINI] ---------- INI ----------
[2020/07/28 19:34:24.914999][I][readINI] StartButton_ON = False
[2020/07/28 19:34:24.917967][I][readINI] PowerSupplier = 66321B
[2020/07/28 19:34:24.919973][I][readINI] PowerSupplierAddr = GPIB0::5::INSTR
[2020/07/28 19:34:24.922015][I][readINI] DigitalMeter = 8261A
[2020/07/28 19:34:24.924991][I][readINI] DigitalMeter_COMPort = 20
[2020/07/28 19:34:24.927993][I][readINI] ELoadTestState = False
[2020/07/28 19:34:24.980015][I][readINI] Voltage_Ch = 107
[2020/07/28 19:34:24.988045][I][readINI] Voltage_Max = 0.3
[2020/07/28 19:34:24.996025][I][readINI] Voltage_Min = 0
[2020/07/28 19:34:25.005089][I][readINI] DetectVoltage = 0.3
[2020/07/28 19:34:25.013110][I][readINI] TestTimeout = 30.0
[2020/07/28 19:34:25.021083][I][readINI] LeastTestTime = 2.5
[2020/07/28 19:34:25.029106][I][readINI] EC = 010000S03
[2020/07/28 19:34:25.035276][I][readINI] ---------- INI ----------
[2020/07/28 19:34:25.039992]---------- startTest ----------
[2020/07/28 19:34:25.043047]================ Control DM ================
[2020/07/28 19:34:25.049020]==================Setup digital meter==================
[2020/07/28 19:34:25.064754][I][setupDigitalMeter] Get visa resource manager...
[2020/07/28 19:34:25.073173][E][setupDigitalMeter] Connect to digital meter Error
[2020/07/28 19:34:25.076184][E][startTest] Control digital meter failed!
[2020/07/28 19:34:25.080196][I][TestResult] Test Fail! Control digital meter failed!
